packages/teleport/src/Databases/Databases.tsx

```diff
@@ -22,13 +22,15 @@ import {
   FeatureHeader,
   FeatureHeaderTitle,
 } from 'teleport/components/Layout';
+import useTeleport from 'teleport/useTeleport';
+import { Attempt } from 'shared/hooks/useAttemptNext';
```

Comment:
since we have a custom hook, that exports fields needed by our main component, we can use the return type of our custom hook as our prop now